SparkPost/2web2ui
# FE-1073 - Cypress tests should pass for both the original and Hibana versions of the app

[FE-1073](https://jira.int.messagesystems.com/browse/FE-1073)

### What Changed
- Updated Cypress tests to pass in both the Hibana and OG themes
- Invalidates CircleCI cache to resolve problem with unpackaging tarball
- Disabled `no-matchbox-import` ESLint rule within the `/cypress` dir
- Renamed `cy.assertLink()` => `cy.verifyLink()` - a more accurate name!
- Fixed env variable detection in Cypress that was broken following a prior refactor
- Resolved a bug for within the `ReportOptions` component in which a Redux action was not properly passed to child components. Unclear as to why this was working at all in the OG theme and not Hibana...

### How To Test
- Verify both sets of tests are passing in CI
- Verify I didn't introduce any problems with updated tests
- Verify I didn't introduce bugs as a result of my updates (particularly by verifying the filtering on `/reports/summary`)

### To Do
